lib/merch_calendar/fiscal_year_calendar.rb

```diff
@@ -1,118 +1,118 @@
-require "merch_calendar/retail_calendar"
-
 module MerchCalendar
   class FiscalYearCalendar
```

Comment:
I might suggest renaming this class to `StitchFixFiscalYearCalendar`.  Its purpose is now very specific to our internal uses.  I think it's still okay to stay in the open source gem, but consumers should be warned that its our personal interpretation of the retail calendar.